src/aatoolbox/utils/codab.py

```diff
@@ -0,0 +1,43 @@
+"""Retrieve COD administrative boundaries."""
```

Comment:
Details, but I would add a bit more explanation / link to what the CODs are

Reply:
Good point! To do for when we expand this class